frontend/src/api/pipelines/__tests__/errorUtils.spec.ts

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable camelcase */
+import { handlePipelineFailures } from '~/api/pipelines/errorUtils';
+import { mockPipelineKF } from '~/__mocks__/mockPipelineKF';
+import { proxyGET } from '~/api/proxyUtils';
+import { NotReadyError } from '~/utilities/useFetchState';
+
+jest.mock('~/api/proxyUtils.ts', () => ({
+  proxyGET: jest.fn(),
+}));
+
+jest.spyOn(console, 'error').mockImplementation(jest.fn());
+
+const proxyGETMock = jest.mocked(proxyGET);
+const host = 'test';
+const path = '/test';
+
+describe('handlePipelineFailures', () => {
+  it('should successfully return pipeline', async () => {
+    const pipelineMock = mockPipelineKF({});
+    proxyGETMock.mockResolvedValue(pipelineMock);
+    const result = await handlePipelineFailures(proxyGETMock(host, path));
+    expect(result).toStrictEqual(pipelineMock);
+    expect(proxyGETMock).toHaveBeenCalledTimes(1);
```

Comment:
Unnecessary to mock this and assert the mock was called when you've executed the function in the test itself.